### DISCUSSION TITLE
WWII Historical Bias Intertwined with Copilot Dialog #127983...

### DISCUSSION THREAD

**Discussion by JimmySutton** [community/community#128224] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

WWII Historical Bias Intertwined with Copilot Dialog https://github.com/orgs/community/discussions/127983... Was This Too Much Dialog for Acknowledgement or Was There Another Reason That There Was No Response To My Feedback?

**Comment by ebndev** [comment]
Hi @JimmySutton, thanks for being a part of the GitHub Community!

I am going to close this discussion as it is a duplicate of [this one](https://github.com/orgs/community/discussions/127983), where another moderator has responded to your post.

Please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service) if you have any questions about cross-posting!
